Added initial UI: sidebar, menu, etc.

waterwo2 opened the pull request:
Added in initial ui components, sidebar, small css, header. Image inclusion and functionality not included in this story.
Jira Story: https://tim-kaufenberg.atlassian.net/browse/ARC-55

Commit:
Initial ui changes

Commit:
json files

Commit:
Adding material-ui components

coughenberg commented:
Please add a description to this pr of what you edited, the Jira story this is referencing, and change the title to be more specific on what is being changed per https://github.com/ARC-Summer-2020/AutoRockClimber/wiki/Pull-Review-Expectations

mattshoobs commented on src/App.tsx:
```diff
@@ -1,5 +1,238 @@
+// import ListItemIcon from '@material-ui/core/ListItemIcon';
+// import ListItemText from '@material-ui/core/ListItemText';
+// import InboxIcon from '@material-ui/icons/MoveToInbox';
+// import MailIcon from '@material-ui/icons/Mail';
```
Remove commented out imports that we are not using.

mattshoobs commented on src/App.tsx:
```diff
@@ -1,5 +1,238 @@
+  drawerHeader: {
+    display: 'flex',
+    alignItems: 'center',
+    padding: theme.spacing(0, 1),
```
Fine for now, once we display an image I think we will need to increase this or come up with a different solution.

mattshoobs commented on src/App.tsx:
```diff
@@ -1,5 +1,238 @@
+          enforcement need. In the makeup, Whosoever shall not nibh sed ac hendrerit gravida ac. Nor for any 
+          sem et tortor Pellentesque. They inhabit the sad old age and disease. Importantly developer homework 
+          is sad more items. Default jasmine Performance element only one deductible. Please pull Maecenas layer 
+          tanks or antioxidants. There is no set clinical care of some avenging arrows.
```
Remove any old commented out code to clean up the file. We wont need the placeholder text.

mattshoobs commented on src/App.tsx:
```diff
@@ -1,5 +1,238 @@
+              <ListItemIcon>{index % 2 === 0 ? <InboxIcon /> : <MailIcon />}</ListItemIcon>
+              <ListItemText primary={text} />
+            </ListItem>
+          ))}
```
Remove old commented out code

mattshoobs reviewed (changes requested):
Clean up by removing old and unused commented out code then it looks good.

Commit:
Merge branch 'arc-ui-sidebar' of https://github.com/ARC-Summer-2020/AutoRockClimber into arc-ui-sidebar

Commit:
Removed excess commented out code

Commit:
Fixed up branch, see below

Kept PR fix to remove extra comments, reverted app to running state with UI menu and sidebar, added missing package.json items.

Commit:
Merge branch 'arc-ui-sidebar' of https://github.com/ARC-Summer-2020/AutoRockClimber into arc-ui-sidebar
